README.md

```diff
@@ -200,6 +200,8 @@ Actions supported:
 * `client.databases.get_eviction_policy(id: 'id')`
 * `client.databases.set_sql_mode(database_sql_mode, id: 'id')`
 * `client.databases.get_sql_mode(id: 'id')`
+* `client.databases.get_metrics_credentials`
+* `client.databases.update_metrics_credentials`
```

Comment:
One small change – the `update_metrics_credentials` takes an argument, so we need to include that in the README docs:
```suggestion
* `client.databases.update_metrics_credentials(database_metrics_credentials)`
```

Reply:
My bad, missed this one!